tests/optional.py

```diff
@@ -21,7 +21,12 @@
 from typing import FrozenSet, List, Set, TYPE_CHECKING
 
 import pytest
-from _pytest.store import StoreKey
```

Comment:
Additionally it would be good to replace uses of `_store` with `stash`. The former still works but will probably be removed at some point.

Reply:
config._store with config.stash?

Reply:
Yes